dashboard/app/models/peer_review.rb

```diff
@@ -142,9 +142,25 @@ def self.create_for_submission(user_level, level_source_id)
     end
   end
 
+  def self.get_review_completion_status(user, script)
+    if user &&
+        script.peer_reviews_to_complete.try(:>, 0) &&
+        Plc::EnrollmentUnitAssignment.exists?(user: user, plc_course_unit: script.plc_course_unit)
+      reviews_done = PeerReview.where(reviewer: user, script: script, status: [0, 1, 2]).size
```

Comment:
Aren't `[0,1,2]` all the [statuses](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/peer_review.rb#L38-L42)? Is this part needed? And if so, is there a way to specify the symbol names instead of the #s here?